### ISSUE TITLE
Release notes report: issues that need fixing

### ISSUE THREAD

**Issue by viomunoz** [mpi2/PhenotypeData#296]
Data relase notes are getting full exposure with the new homepage style being proposed (see image below) and thus the team is working to make improvements to them.

### Requirements:
1. - [x] Unambiguously indicate the version to reference the genome we are using.
The genome version we are using is reported as "GRCm38". The current one is GRCm38.p6 (Genome Reference Consortium Mouse Build 38 patch release 6, released Sep 15 2017). So as a user I may wonder which version it is that is really being used. One way to solve this would be to reference the assembly accession id (Assembly accessions: GenBank: [GCA_000001635.8](https://www.ebi.ac.uk/ena/data/view/GCA_000001635.8), for GRCm38.p6; Assembly accessions: GenBank: [GCA_000001635.2](https://www.ebi.ac.uk/ena/data/view/GCA_000001635.2), for GRCm38) and the URL from which it was downloaded (I guess we use Ensembl, but there are other sources).
**_@mrelac-ebi:  Where/how do we get this?_**
**_@viomunoz  We learnt more about this since this ticket was written. We get this data from MGI http://www.informatics.jax.org/downloads/reports/index.html#seq. An option is to leave it as is and the interested user will ask_**
**_@mrelac-ebi and @viomunoz decided to Insert link to MGI home page https://www.informatics.jax.org_**

1. - [x] Provide reference and URL address to Statistical Package (PhenStat).
**_@mrelac-ebi Get this from @Hamedhm ? Jeremy?_**
**_@viomunoz  There have been some changes since this ticket was written. We need to reference OpenStat now: https://www.biorxiv.org/content/10.1101/2020.05.13.091157v1 (@jmason-ebi to confirm). We could also add a link to the Statistics Section in the Documentation (following Lynn's advice to avoid dead ends): https://www.mousephenotype.org/help/data-analysis/_**
**_@mrelac-ebi and @viomunoz decided to link to the statistical analysis help page_**

1. - [x] Make sure FTP link redirects to the FTP documentation, here: (https://www.mousephenotype.org/help/non-programmatic-data-access/)

1. - [x] Make sure API link redirects to the API documentation, here: (https://www.mousephenotype.org/help/programmatic-data-access/)

1. - [x] "Lines and Specimens" perhaps better change to "Total Number of Lines and Specimens in DR 11.0".

1. - [x] Indicate what is being counted in "Experimental Data and Quality Checks".
**_@mrelac-ebi This is a count, by IMPC data type, of the toal number of datapoints that passed QC. Its value is taken from this query: `select * from meta_info where property_key like '%QC_passed'`_**
**_@mrelac-ebi  and @viomunoz agreed: Change 'QC passed' label to 'QC passed datapoints'_**

1. - [x] Indicate whether "Distribution of Phenotype Annotations" are counts per gene or allele (mutant lines); users may download and use this graph, so good they know.
**_@mrelac-ebi This information comes from this g-p core query: http://ves-ebi-d0.ebi.ac.uk:8986/solr/genotype-phenotype/select?q=zygosity:heterozygote&facet=true&facet.limit=-1&facet.mincount=1&rows=0&facet.field=top_level_mp_term_name_**
**_@mrelac-ebi  and @viomunoz agreed to leave the y-axis label as is and to try inserting an x-axis label 'Top-level MP term' if it looks good. Don't spend too much time if the label doesn't render nicely. The idea is to provide an x-axis label if easily doable._**
**_@mrelac-ebi: @jmason-ebi  suggested updating the chart subtitle to 'Distribution of Phenotype Associations by Top-level MP Term'_**

1. - [x] Change header "Status" to "Production Status".

1. - [x] Users may not know what iMits is. Change "More charts and status information are available from iMits." to "More charts and status information are available from the IMPC mouse production tracking system, [iMits](https://www.mousephenotype.org/imits/v2/reports/mi_production/komp2_graph_report_display) (for information about iMits, please see our [Documentation] (https://www.mousephenotype.org/help/mouse-production/production-tracking-imits/) )."
**_@mrelac-ebi and @viomunoz agreed to change the text to 'More charts and status information are available from our mouse-tracking services iMits and GenTaR'. iMits links to https://www.mousephenotype.org/imits and GenTaR to https://www.gentar.org/_**

1. - [x] Section "Phenotype Associations": procedure names will be meaningless to most users. Do we want to keep this figure (eg. useful for centers)?
**_@mrelac-ebi  and @viomunoz and @jmason-ebi met 19-Oct-2020 and decided to:_**
        - replace procedure stable ids with procedure names
        - Replace grouping by center with grouping by:
            - embryo procedures (sorted by life stage, then procedure name)
            - non-embryo procedures (sorted by procedure name)

1. - [x] Section "Trends" x-axis label missing: "Data Release".
**_@mrelac-ebi: @jmason-ebi and @viomunoz suggested replacing the chart subtitle 'Release by Relase' with 'By Data Release'_**

1. - [x] Legacy release pages are slow to load. Can we speed them up?

**Comment by mrelac-ebi** [comment]
15-07-2020: Minor fixes from above:
  Fix ftp link (#4)
  Fix api link (#5)
  Fix 'Lines and Specimens' (#6)
  Fix 'Status' (#9)

**Comment by Hamedhm** [comment]
Wow impressive work!

Provide reference and URL address to Statistical Package (PhenStat).  see two URLs below (one is already noted by @viomunoz):

- http://www.bioconductor.org/packages/release/bioc/html/OpenStats.html
- https://www.biorxiv.org/content/10.1101/2020.05.13.091157v1


Indicate what is being counted in "Experimental Data and Quality Checks".

- I do not know but I think it comes from wranglers


Users may not know what iMits is. Change "More charts and status information are available from iMits." to "More charts and status information are available from the IMPC mouse production tracking system, iMits (for information about iMits, please see our Documentation )."
@mrelac-ebi iMits? genTar? Is the link still co

- I am always lean toward less using acronyms

Indicate whether "Distribution of Phenotype Annotations" are counts per gene or allele (mutant lines); users may download and use this graph, so good they know.
- I guess it is per colony

**Comment by mrelac-ebi** [comment]
@jwgwarren is familiar with the highcharts helpers and will investigate to see if x-axis labels and units can be easily added.

**Comment by mrelac-ebi** [comment]
Done. Regarding the previous comment, there was no need for x-axis labels, as we described the content in the title.
